main.go

```diff
@@ -72,3 +72,55 @@ func stripPath(path string) string {
 	dir := filepath.Dir(path)
 	return strings.TrimPrefix(dir, build.Default.GOPATH+"/src/")
 }
+
+// safeRename safely moves files across different file systems.
```

Comment:
I'm not really "married" to the name. The reason I called it safeRename is that it wraps the existing os.Rename() std function. If moveFile() is preferred, I'll rename it.